stack.yaml

```diff
@@ -1,4 +1,4 @@
-resolver: lts-8.12
+resolver: lts-8.13
```

Comment:
One of the symptoms of the bug is that if you write `stack install happy` with LTS 8.12, it will fail to copy the binary.